marge/batch_job.py

```diff
@@ -225,6 +237,11 @@ def execute(self):
         )
         batch_mr_sha = batch_mr.sha
 
+        # delay in case Gitlab is slow to respond
+        waiting_time_in_secs = 20
+        log.debug('Waiting for %s secs for MR !%s to get status update', waiting_time_in_secs, batch_mr.iid)
+        time.sleep(waiting_time_in_secs)
```

Comment:
Waiting for 20 seconds to get status update seems brutal. Would it be better to have a function to poll the status and have an interval and a limit? It can be used for other sleeps as well.